Jenkinsfile

```diff
@@ -42,10 +42,149 @@ pipeline {
               newImage.push('latest')
             }
           }
+          if (env.TAG_NAME == 'production-release') {
+            stage('Update production release tag') {
+              newImage.push('production-release')
+            }
+          }
+        }
+      }
+    }
+
+    stage('Build AMIs') {
+      when { tag 'production-release' }
+      failFast true
+      parallel {
+        stage('Production API') {
+          options {
+            skipDefaultCheckout true
+          }
+          agent {
+            docker {
+              image 'zooniverse/operations:latest'
+              args '-v "$HOME/.ssh/:/home/ubuntu/.ssh" -v "$HOME/.aws/:/home/ubuntu/.aws"'
+            }
+          }
+          steps {
+            sh """#!/bin/bash -e
+              while true; do sleep 3; echo -n "."; done &
+              KEEP_ALIVE_ECHO_JOB=\$!
+              cd /operations
+              ./rebuild.sh -c panoptes-api
+              kill \${KEEP_ALIVE_ECHO_JOB}
+            """
+          }
+          post {
+            success {
+              slackSend (
+                  color: '#00FF00',
+                  message: "SUCCESSFUL: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
+                  channel: "#ops"
+                  )
+            }
+            failure {
+              slackSend (
+                  color: '#FF0000',
+                  message: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
+                  channel: "#ops"
+                  )
+            }
+          }
+        }
+        stage('Production Dump workers') {
+          options {
+            skipDefaultCheckout true
+          }
+          agent {
+            docker {
+              image 'zooniverse/operations:latest'
+              args '-v "$HOME/.ssh/:/home/ubuntu/.ssh" -v "$HOME/.aws/:/home/ubuntu/.aws"'
+            }
+          }
+          steps {
+            sh """#!/bin/bash -e
+              while true; do sleep 3; echo -n "."; done &
+              KEEP_ALIVE_ECHO_JOB=\$!
+              cd /operations
+              ./rebuild.sh -c panoptes-dumpworker
+              kill \${KEEP_ALIVE_ECHO_JOB}
+            """
+          }
+          post {
+            success {
+              slackSend (
+                  color: '#00FF00',
+                  message: "SUCCESSFUL: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
+                  channel: "#ops"
+                  )
+            }
+            failure {
+              slackSend (
+                  color: '#FF0000',
+                  message: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
+                  channel: "#ops"
+                  )
+            }
+          }
         }
       }
     }
 
+    stage('Deploy production AMIs') {
```

Comment:
I don't think we want this deploy script to automatically run on `production-release` tag (migrations). We will want to run the deploy via the lita cmd as per the previous setup.

Reply:
My aim is to have this deploy using the newer `lita deploy` command, which works by updating the tag to trigger the deploy. I'm not trying to get the old `lita panoptes deploy` command working again.

Reply:
our current issue is running migrations as they will need to run before the code deploys to ensure the db schema matches the code ORM schema. If we can finalize the migration jobs running correctly (on their own tag) then i think we can safely pursue this deploy path, i.e. 
1. merge to master deploy to staging
2. run a migration job by updating `production-migrate` tag
3. update the `production-release` tag to trigger the prod deployment